app/views/tasks/_in_progress_tasks.html.erb

```diff
@@ -7,7 +7,7 @@
       <% tasks.each do |task| %>
           <ul class="list-group">
             <li class="list-group-item">
-              <%= task.name %>
+              <%= link_to (task.name), project_task_path(@project,task) %>
```

Comment:
the same as [above](https://github.com/dKotetunov/TasksManager/pull/12/files#r29217728)